providers/aws/flags.go

```diff
@@ -0,0 +1,63 @@
+package aws
+
+import "github.com/urfave/cli/v2"
+
+const Category = "AWS"
+
+var DriverFlags = []cli.Flag{
+	// aws
+	&cli.StringFlag{
+		Name:     "aws-instance-type",
+		Usage:    "aws instance type",
+		EnvVars:  []string{"WOODPECKER_AWS_INSTANCE_TYPE"},
+		Category: Category,
+	},
+	&cli.StringFlag{
+		Name:     "aws-ami-id",
+		Usage:    "aws ami id",
+		EnvVars:  []string{"WOODPECKER_AWS_AMI_ID"},
+		Category: Category,
+	},
+	&cli.StringSliceFlag{
+		Name:     "aws-tags",
+		Usage:    "aws tags",
+		EnvVars:  []string{"WOODPECKER_AWS_TAGS"},
+		Category: Category,
+	},
+	&cli.StringFlag{
+		Name:     "aws-region",
+		Usage:    "aws region",
+		EnvVars:  []string{"WOODPECKER_AWS_REGION"},
+		Category: Category,
+	},
+	&cli.StringSliceFlag{
+		Name:     "aws-subnets",
+		Usage:    "aws subnets",
```

Comment:
Not 100% getting what value I would set here. Could you adjust the usage / description value and add an example to it.